src/BaseVideoNodeTile.h

```diff
@@ -12,7 +12,7 @@ typedef QmlSharedPointer<VideoNode> VideoNodeSP;
 class BaseVideoNodeTile : public QQuickItem {
     Q_OBJECT
     Q_PROPERTY(ModelSP *model MEMBER m_model NOTIFY onModelChanged);
-    Q_PROPERTY(VideoNodeSP *videoNode MEMBER m_videoNode NOTIFY onVideoNodeChanged);
+    Q_PROPERTY(VideoNodeSP *videoNode MEMBER m_videoNode READ videoNode NOTIFY onVideoNodeChanged);
```

Comment:
Was this reasonable to do? I wanted to go from tile -> `VideoNodeSP *` in View.cpp and I didn't want to use the generic property + cast because I wanted to make sure the types worked at compile time (it's one thing to cast an int but this has multiple layers of pointer-ness)

Reply:
yeah this is fine. you may want to just add a WRITE method, having a getter but no setter is sort of strange given that both getting and setting are allowed.

Reply:
Eh, the setter is still exposed through the property system (MEMBER)